cmd/ntpcheck/cmd/utils.go

```diff
@@ -108,13 +116,15 @@ func ntpDate(remoteServerAddr string, remoteServerPort string, requests int) err
 		return err
 	}
 
-	fmt.Printf("Server: %s, Requests: %d\n", addr, requests)
-	var sumAvgNetworkDelay int64
+	var sumDelay int64
 	var sumOffset int64
+	var clientTransmitTime time.Time
```

Comment:
Whats the point of a separate definition?

Reply:
clientTransmitTime and clientWireTransmitTime are not guaranteed to be same.

Reply:
That's not the question :) Question is why you need to move the definition and not in the loop. Or rather why change this part at all?

Reply:
Missed that. Yes, make sense you do.